### PR TITLE
Fix typos and improve grammar for aesthetics

### PR DESCRIPTION
**Pull request by ATechnoHazard**
Done as a part of Google Code-in 2017, Cheers!

### CODE REVIEW

File: docs/installing_dependencies.adoc [hunk 1]
```diff
@@ -46,8 +46,8 @@ To install g2p-seq2seq :
     sudo python setup.py install
 
 
-*I have incorporated above steps into a bash script located inside `install/` directory, that you may simply call using :*
+*I have incorporated above the steps into a bash script located inside `install/` directory. You may call it simply by using :*
```
Comment (canihavesomecoffee) [inline docs/installing_dependencies.adoc]: I think that you should swap around "above" and "the steps", that makes more sense :)

**Comment by ATechnoHazard** [comment]
Oh damn, I did this in a jiffy, needed to build my project with the latest
commits, I'll fix and create a new PR asap

**Comment by canihavesomecoffee** [comment]
@ATechnoHazard you can also push new commits to this pull request :)

**Comment by saurabhshri** [comment]
@ATechnoHazard As @canihavesomecoffee you can make a push commits to this branch and that will automatically be added to this PR. You don't need to make a separate PR. I'll wait for the changes recommended by Willem and merge once you add those.

**Comment by ATechnoHazard** [comment]
Yeah, I was kinda busy with a few build errors on my ROM, I'll push in some
time

**Comment by ATechnoHazard** [comment]
Done!

**Comment by saurabhshri** [comment]
Nice! Thanks for the PR! :)

**Comment by ATechnoHazard** [comment]
Sure thing👍

**Pull request merged**